install_config/configuring_sdn.adoc

```diff
@@ -164,6 +164,181 @@ Check VNIDs by running:
+`*networkConfig*` stanza of the *_master-config.yaml_* file, and then
+restart the master; for single master clusters:
+
+----
+# systemctl restart atomic-openshift-master
+----
+
+For multi-master clusters:
+
+----
+# systemctl restart atomic-openshift-master-controllers
+----
+
+You will also need to restart the node service on the master and on
+each of the nodes, to make them update their IP routing information to
+take the larger cluster network into account:
+
+----
+# systemctl restart atomic-openshift-node
+----
+
+Note that this only works if you are changing `*clusterNetworkCIDR*`
+to a new value which is larger than the old value, and which includes
+the old value as a subset; if you try to change it to any other value,
+then when you restart the master, it will complain about the mismatch
+and refuse to start. You will need to use the procedure below in this
+case.
+
+=== Other Changes to the Cluster Network IP Range
+
+If you want to completely change `*clusterNetworkCIDR*` (eg, moving
+from *10.128.0.0/14* to *192.168.9.0/8*) or you want to change
+`*hostSubnetLength*` (to allow more or fewer pods per node), you will
+need to completely take down the cluster temporarily.
+
+. On every node and master, do:
++
+----
+# systemctl stop atomic-openshift-node
+# systemctl restart docker
+----
+
+. As a cluster admin user, run:
++
+----
+$ oc delete hostsubnets --all
+$ oc delete clusternetwork default
+----
+
+. Stop the masters. In a single-master cluster:
++
+----
+# systemctl stop atomic-openshift-master
+----
++
+For multi-master clusters:
++
+----
+# systemctl stop atomic-openshift-master-controllers
+# systemctl stop atomic-openshift-master-api
+----
+
+. Now edit the *_master-config.yaml_* file and make the desired changes.
+
+. Restart the masters:
++
+----
+# systemctl start atomic-openshift-master
+----
++
+or (multi-master):
++
+----
+# systemctl start atomic-openshift-master-controllers
+# systemctl start atomic-openshift-master-api
+----
+
+. Restart the node service on all of the nodes and masters:
++
+----
+# systemctl start atomic-openshift-node
+----
+
+At this point the cluster will be back up, but every
+previously-running pod will have been killed, and it may take a few
+minutes to get everything completely back up and running again.
+
+=== Changing the Service Network IP Range
+
+If you need to change `*serviceNetworkCIDR*`, either because you need
+more than 65,536 service IPs, or because you have belatedly discovered
+a conflict between the service IP range and other IPs on your network,
+you will have to take down the cluster (as above), but you will also
+need to delete and recreate all Service objects. There is no simple
+way to do this, and in particular, to get all of the permissions
+correct, each Service will have to be recreated by the correct user;
+you can't simply have an administrator recreate them. Thus, this will
+result in a more noticeable outage than the `*clusterNetworkCIDR*`
+renumbering case.
```

Comment:
Is it worth mentioning that as each service is recreated it will get a new address from the new serviceNetworkCIDR? There may be other required changes in the apps when the service ip changes.